aspnetcore/signalr/webpack-and-typescript.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Using ASP.NET Core SignalR with TypeScript and WebPack
+author: ssougnez
+description: How to bundle a web application using SignalR, TypeScript and WebPack.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: ssougnez
+ms.custom: mvc
+ms.date: 06/13/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/webpack-and-typescript
+---
+
+# Using ASP.NET Core SignalR with TypeScript and WebPack
+
+By [Sébastien Sougnez](https://twitter.com/ssougnez)
+
+**WebPack** enables developers to easily bundle the client side of a web application. This tutorial teaches how to use it to bundle an application written in **TypeScript** that communicates with a server through web sockets using **SignalR**.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/live/aspnetcore/signalr/webpack-and-typescript/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample))
+
+## Prerequisites
+
+* **[NodeJs](https://nodejs.org/en/)** and **npm** must be installed on your system to be able to use **WebPack**.
+
+There is two options to run **WebPack** with **Visual Studio**:
+
+* Either you use an external terminal.
+* Or you can use the **WebPack Task Runner** extension that allows you to see the output of **WebPack** directly in **Visual Studio** and launch the compilation automatically when the project is opened. If you opt for this option, please follow these instructions:
```

Comment:
Let's not mention WebPack Task Runner in this doc. It hasn't kept up with the WebPack releases and is an optional component. For Visual Studio instructions, can you use the Package Manager Console window instead?

Reply:
Okay but damienbod asked me to mention "Visual Studio Task Runner", which is why I added it. As far as I know, the reguler Visual Studio Task Runner does not support WebPack => the extension. 

So I guess you mean that I should run the npm scripts by using the Package Manager Console ?

Reply:
You're correct that the Task Runner Explorer window doesn't natively support WebPack. By removing references to the WebPack Task Runner extension, this document becomes much more future proof and easier for us to maintain. npm scripts in the Package Manager Console window are my recommendation.

Reply:
Okay then, I'll work on that soon ;-)

Reply:
Thank you for your help with this, by the way.